SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -56,10 +67,23 @@ github.com/containers libraries, such as Buildah, CRI-O, Podman and Skopeo.
 
 %prep
 %setup -q -T -D -b 0 -n image-%{imagever}
+# NOTE: Patch3 has to be applied as -p6
+%patch 3 -p6
+
 %setup -q -T -D -b 1 -n storage-%{storagever}
+
 %setup -q -T -D -b 7 -n podman-%{podmanver}
+%patch 0 -p1
+%patch 1 -p1
+%patch 2 -p1
+%patch 3 -p1
+%patch 4 -p1
+
 %setup -q -T -D -b 9 -n common-%{commonver}
 # copy the LICENSE file in the build root
+%patch 0 -p1
+%patch 1 -p1
+%patch 3 -p1
 cd ..
 cp %{SOURCE2} .
```

Comment:
Nic-pick: I think this license comment got misplaced after the patching got added.:)
```suggestion
%setup -q -T -D -b 9 -n common-%{commonver}
%patch 0 -p1
%patch 1 -p1
%patch 3 -p1
cd ..


# copy the LICENSE file in the build root
cp %{SOURCE2} .
```